packages/eslint-config-skuba/package.json

```diff
@@ -34,10 +34,13 @@
     "typescript": "~5.3.0"
   },
   "peerDependencies": {
-    "eslint": ">= 7.2",
-    "typescript": ">= 3.9"
+    "eslint": ">=8.56.0",
+    "typescript": ">=4.7.5"
   },
   "packageManager": "pnpm@8.15.5",
+  "engines": {
+    "node": ">=18.18.0"
```

Comment:
are there any templates that need updating to this new minimum as well?

Reply:
Nay, all templates are on node 20 already